client/my-sites/theme/main.jsx

```diff
@@ -890,11 +890,7 @@ class ThemeSheet extends Component {
 				<UpsellNudge
 					plan={ PLAN_BUSINESS }
 					className="theme__page-upsell-banner"
-					title={
-						siteId
-							? translate( 'Upgrade your plan to access third-party themes' )
-							: translate( 'Access third-party themes with a Business plan' )
-					}
+					title={ translate( 'Access this third-party theme with the Business plan!' ) }
```

Comment:
I was surprised to see that final exclamation mark (`!`). Is that intentional? Just asking because I think we don't use that tone very often (at least on nudges).

Reply:
Never mind, I didn't notice the previous nudge also had the exclamation mark 🤦. I was comparing just the changes from the latest commits, which is why I didn't see the original text:
<img width="1699" alt="Screenshot 2023-02-01 at 10 15 16" src="https://user-images.githubusercontent.com/1233880/216001059-c7781a4a-0885-4ade-9d82-e9147ac351d3.png">